doc: fix extname with the correct description (#27303)
nodejs/node: doc/api/path.md

The `path.extname()` method returns the extension of the `path`, from the last
occurrence of the `.` (period) character to end of string in the last portion of

> the `path`. If there is no `.` in the last portion of the `path`, or if 

Reviewer: The linter is complaining about the trailing space on this line. 

```suggestion
the `path`. If there is no `.` in the last portion of the `path`, or if
```

> there are no dots after the first character of

Reviewer: ```suggestion
there are no `.` characters other than the first character of
```
